cli/new.md

```diff
@@ -13,10 +13,13 @@ Arguments:
   NAME  name/path of project
 
 Options:
-  -d      Select the database database engine, can be one of: pg | mysql | sqlite (default: pg)
-  --deps  Installs project dependencies, this is the equivalent of running (shards update)
-  -t      Selects the template engine language, can be one of: slang | ecr (default: slang)
-  -r      Use a named recipe.  See documentation at https://amberframework.org.
+  -d          Select the database database engine, can be one of: pg | mysql | sqlite (default: pg)
+  --deps      Installs project dependencies, this is the equivalent of running (shards update)
+  -m          Select the model type, can be one of: granite | crecto (default: granite)
+  --no-color  Disable colored output
+  -t          Selects the template engine language, can be one of: slang | ecr (default: slang)
+  -h, --help  Show this help
+  -r          Use a named recipe.  See documentation at https://amberframework.org.
```

Comment:
I think we can use https://amberframework.gitbook.io/amber/cli/recipes here:

```
-r          Use a named recipe.  See documentation at https://amberframework.gitbook.io/amber/cli/recipes.
```

We may need to update it on amber repo itself, see: https://github.com/amberframework/amber/blob/ba70289c99aaae10fd141e65a27d91bed67b533c/src/amber/cli/commands.cr#L51

Reply:
ok 👍 I will create a pull request on amber repo.

Reply:
Hi, I think we can fix that later :sweat_smile: 

I already opened an issue about this, see:https://github.com/amberframework/amber/issues/833

I gonna merge this :+1: